cmd/kyma/alpha/list/module/module.go

```diff
@@ -214,12 +214,16 @@ func (cmd *command) printKymaActiveTemplates(ctx context.Context, kyma *unstruct
 		if err != nil {
 			return err
 		}
-		anns := tpl.GetAnnotations()
-		if anns == nil {
-			anns = make(map[string]string)
+		annotations := tpl.GetAnnotations()
+		if annotations == nil {
+			annotations = make(map[string]string)
 		}
-		anns["state.cmd.kyma-project.io"] = item["state"].(string)
-		tpl.SetAnnotations(anns)
+		stateValue, ok := item["state"].(string)
+		if !ok {
+			continue
+		}
```

Comment:
It looks here you're using type assertion to verify if the map entry exists... If it doesn't, `item["state"]` resolves to `nil` and then the type assertion fails. IMO this is slower and less readable than explicit handling, like this:
```suggestion
		stateValue, exists := item["state"]
		if !exists {
			continue
		}
		stateValueString, ok := stateValue.(string)
		if !ok {
			continue
		}
```

Reply:
Nice find, split